ttnn/cpp/ttnn/tensor/shape/shape.hpp

```diff
@@ -30,6 +30,8 @@ class SimpleShape final : protected ShapeBase {
     [[nodiscard]] size_t rank() const;
     [[nodiscard]] uint64_t volume() const;
 
```

Comment:
No need to return const value here. 

Why is this API needed?

Reply:
I added it for convenience since it's a commonly used helper for legacy shape shape. It makes it easier to switch to simpleshape. I can move it elsewhere if that's preferred.